Silicon/CoffeelakePkg/Library/FirmwareUpdateLib/FirmwareUpdateLib.c

```diff
@@ -131,6 +133,8 @@ InitCsmeUpdInputData (
 {
   CSME_UPDATE_DRIVER_INPUT    *CsmeUpdDriverInput;
 
+  mHeciService = (HECI_SERVICE *)GetServiceBySignature (HECI_SERVICE_SIGNATURE);

```

Comment:
This can be a local variable.

Reply:
Modified to local variable.

Reply:
Can you remove 'm' prefix for the local variable?

Reply:
Done